ui/shared/utils/localStorage.js

```diff
@@ -8,7 +8,9 @@ export const saveState = (label, state) => {
     const serializedState = JSON.stringify(state)
     localStorage.setItem(label, serializedState)
   } catch (err) {
-    // Ignore write errors.
+    console.warn(err) // eslint-disable-line no-console
```

Comment:
Since we need the error message from our users and they don't usually use the debugging tool to check the message, why don't we use the `window.alert()` to show the error message to the end users?

Reply:
because I do NOT want end users to see this, I just want the ability for me to see it if I need to dive into debugging an issue that this causes